bookdown/index.Rmd

```diff
@@ -15,6 +14,12 @@ cover-image: "block.png"
 favicon: "favicon.ico"
 apple-touch-icon: "apple-touch-icon.png"
 apple-touch-icon-size: 180
+# 'geometry' has only an effect when set here, not in output.yml
+geometry:
```

Comment:
Why are we doing this via yaml? Wouldn't it be better to put stuff like this in the preamble?

Reply:
Also, I still don't get why using custom margins is better than defaulting to some presets koma script provides (DIV=14).

Reply:
> Why are we doing this via yaml? Wouldn't it be better to put stuff like this in the preamble?

Imo we should use the pandoc way of changing settings rather than using LaTeX code in `preamble.tex`. 

Also some settings are otherwise set twice (pandoc default + preamble.tex) and will cause an error.

> Also, I still don't get why using custom margins is better than defaulting to some presets koma script provides (DIV=14).

I don't know why the defaults should always be the best setting. Most often, they are just defaults...

Reply:
> Imo we should use the pandoc way of changing settings rather than using LaTeX code in `preamble.tex`.

I don't see an advantage if we already have a `preamble.tex`. You can put everything (well, maybe not the documentclass) into the preamble, but you can only put a tiny fraction of latex into the yaml. We should keep settings  together.


> I don't know why the defaults should always be the best setting. Most often, they are just defaults...

... defaults carefully chosen by typographers. KOMA goes back to the work of Jan Tschichold. See his wiki page (https://en.wikipedia.org/wiki/Jan_Tschichold) for considerations on how to determine good margins.

Reply:
> We should keep settings together.

That's what I did following the YAML approach.
Feel free to move it.
But be aware that this can introduce problems for some settings due to conflicts with pandocs template settings.

If you dislike the new margins, feel free to remove them again. 
I will still dislike the typographers defaults ;)